### PR TITLE
Add Links the to base API and SDK

### PR DESCRIPTION
**Pull request by hdost**
Signed-off-by: Harold Dost <h.dost@criteo.com>

### CODE REVIEW

**Comment by hdost** [comment]
The SDK and API were split into separate crates https://github.com/open-telemetry/opentelemetry-rust/pull/735

File: content/en/docs/instrumentation/rust/_index.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ as follows:
 
 Opentelemetry for Rust publishes the following crates:
 - [opentelemetry](https://crates.io/crates/opentelemetry)
+- [opentelemetry-api](https://crates.io/crates/opentelemetry-sdk)
```
Comment (cartermp) [inline content/en/docs/instrumentation/rust/_index.md:28]: should this be `opentelemetry-api`?

Comment (hdost) [reply content/en/docs/instrumentation/rust/_index.md]: done

**Review by cartermp — APPROVED** [review]

**Review by chalin — APPROVED** [review]
Thanks all

**Pull request merged**